src/urbit/app/chess.hoon

```diff
@@ -835,18 +835,66 @@
   ^-  (unit (unit cage))
   ?+  path  (on-peek:default path)
     ::
-    ::  .^(noun %gx /=chess=/game/~1996.2.16..10.00.00..0000/noun)
-    ::  read game info
-    ::  either active or archived
-    [%x %game @ta ~]
+    :: send back a list of chess-games
+    [%x %archive *]
+      ?+  t.t.path  (on-peek:default path)
+      ::
+      ::  =sur -build-file /=chess=/sur/chess/hoon
```

Comment:
I see why you added this but I think anyone who needs to know this knows it already, and the examples here just need to be scry paths.

Reply:
okie dokie.